crates/net/eth-wire-types/src/broadcast.rs

```diff
@@ -765,6 +765,27 @@ impl FromIterator<(TxHash, Eth68TxMetadata)> for RequestTxHashes {
     }
 }
 
+/// This informs peers of the updated block range the node can provide blocks for.
+/// See [EIP-7642](https://eips.ethereum.org/EIPS/eip-7642).
+#[derive(Clone, Debug, PartialEq, Eq, Default, RlpEncodableWrapper, RlpDecodable)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct NewBlockRange {
+    /// New [`BlockRange`].
+    pub block_range: BlockRange,
+}
```

Comment:
let's reference

https://github.com/ethereum/devp2p/blob/master/caps/eth.md#blockrangeupdate-0x11

we can also get rid of the outer `NewBlockRange` and can rename `BlockRange` to `BlockRangeUpdate`